cmd/flux/bootstrap_git.go

```diff
@@ -85,6 +92,18 @@ func init() {
 }
 
 func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
+	if gitArgs.password == "" {
```

Comment:
This change breaks bootstrap for most users, the password is an option field, usually you would run bootstrap using SSH.

Reply:
Because password is optional, I've two suggestions how to fix this:

1. Change the condition to `if gitArgs.username != "" && gitArgs.password == "" {` as password is required when a username is defined. With this option, ssh auth method is not supported (in case password is required for the private-key)
2. If the env var is defined && password is empty, use it. This apply for https and ssh but we won't have a prompt in case the user would like to provide the password as stdin.

IMO, option 2 seems better. Users should not provide the password thru the CLI. 

wdyt?

Reply:
I've updated the commit to handle both cases. If the user specifies `--token-auth` and password is not set, they will be prompted for typing it.